viral/sequence_utils.py

```
@@ -0,0 +1,859 @@
+    point1y = radon_lut.point1y[line_idx, theta_idx]
+    point2x = radon_lut.point2x[line_idx, theta_idx]
+    point2y = radon_lut.point2y[line_idx, theta_idx]
+
+    # TODO: this seems off: check units
+    # distance = ((point2y - point1y) - 0.5) * bayesian_config.bin_size_spatial / 100
+    bin_size_time = (
+        bayesian_config.bin_size_time_online
+        if bayesian_config.online
+        else bayesian_config.bin_size_time_offline
+    )
+    time = ((point2x - point1x) - 0.5) * bin_size_time / 30
```

Ruff (0.14.10)

Local variable `time` is assigned to but never used

Remove assignment to unused variable `time`

(F841)